app/src/renderer/components/staking/PageBond.vue

```diff
@@ -159,7 +153,7 @@ export default {
         try {
           await this.$store.dispatch('submitDelegation', this.fields)
           this.$store.commit('notify', { title: 'Atoms Bonded',
-            body: 'You have successfully bonded your atoms. You can rebond after the  30 day unbonding period.' })
+            body: 'You have successfully updated your delegations.' })
```

Comment:
"delegated your atoms"?
"bonded your atoms"?

Reply:
I figured it might be weird if you only unbond and then see "you have bonded your atoms". What do you think?

Reply:
oh i see! ok. makes sense.